gobblin-modules/gobblin-orc-dep/build.gradle

```diff
@@ -23,7 +23,7 @@
 
 buildscript {
     repositories {
-        jcenter()
+        gradlePluginPortal()
```

Comment:
For whatever reason shadow <= 6.4 uses gradle plugin portal:

source: https://github.com/johnrengelman/shadow/pull/641